vertical-pod-autoscaler/deploy/vpa-v1-crd-gen.yaml

```diff
@@ -17,7 +17,14 @@ spec:
     singular: verticalpodautoscalercheckpoint
   scope: Namespaced
   versions:
-  - name: v1
+  - additionalPrinterColumns:
```

Comment:
Let's not do pretty printing on internal state-storing objects.

Reply:
fix: 73f49e2